packages/kbn-optimizer/limits.yml

```diff
@@ -30,7 +30,7 @@ pageLoadAssetSize:
   data: 454087
   datasetQuality: 50624
   dataViewEditor: 28082
-  dataViewFieldEditor: 27000
+  dataViewFieldEditor: 42017
```

Comment:
Looks like this doesn't need to change

Reply:
It was automatically updates by using the `node scripts/build_kibana_platform_plugins --focus fieldsMetadata --update-limits` script for the fieldsMetadata, I guess it reached a threshold bundle size where the script established it should get that new value.

Reply:
@tonyghiani I think you ran it before some recent changes that reduced the bundle size impact on api consumers. If you revert the change and rerun the command you'll see that its not changed again.

Reply:
[refactor(fields-metadata): revert change to dataViewFieldEditor bundl…](https://github.com/elastic/kibana/pull/183806/commits/0817162997133328a9dcc95435a28b9e4413cb70)